app/models/ability/poll.rb

```diff
@@ -48,5 +48,10 @@ def initialize(user)
     can :close, ::Poll do |poll|
       poll.active? && (user_is_author_of?(poll) || user_is_admin_of?(poll.group_id))
     end
+
+    can :export, ::Poll do |poll|
+      #TODO: review priveledges before merge 
+      can?(:show, poll)
```

Comment:
I'm feeling like we want to limit this to admins and open it up if people complain. We can simply add `:export` to the list of permissions above with :update, :share, :remind, etc

```
can [:update, :share, :remind, :export], ::Poll do
```

Reply:
Part of my reasoning for this is that :show is _very_ relaxed for some polls (since we have polls that anyone can participate in), meaning basically anyone could download the result of the poll.

I'm not really opposed to this (it is accessible, after all), but the only people who have asked for this are people who are coordinating a group, so I want to give this to them first.

Reply:
ok, i'll make it poll authors and admins for now